gems/pending/disk/modules/AzureBlobDisk.rb

```diff
@@ -0,0 +1,91 @@
+require_relative "../MiqDisk"
+require 'ostruct'
+
+module AzureBlobDisk
+  # The maximum read length that supports MD5 return.
+  MAX_READ_LEN = 1024 * 1024 * 4
+
+  def self.new(svc, blob_uri, dInfo = nil)
+    d_info = dInfo || OpenStruct.new
+    d_info.storage_acct_svc = svc
+    d_info.blob_uri         = blob_uri
+    d_info.fileName         = blob_uri
+
+    MiqDisk.new(self, d_info, 0)
+  end
+
+  def d_init
```

Comment:
It came as a surprise to me that there's a .new, d_init, d_close, etc. methods in a module.  I'd expect behavior in modules, not state.  Also, it's a AzureBlobDisk, do we need the d_ prefix?